_includes/markdown/Structure.md

```diff
@@ -197,3 +197,49 @@ indent_size = 4
 ```
 
 Developers may extend and/or customize these rules as new file formats are added to the project.
+
+<h2 id="privacy">Privacy {% include Util/top %}</h2>
+
+Software and services should be designed with privacy in mind, both for user trust and for potential legal reasons. The following are some general guidelines and WordPress-focused features and tactics to use. Accommodations for specific regulations such as GDPR should be determined on a case-by-case basis.
+
+Broadly, the following areas should be considered when designing for privacy:
+
+- What user data is collected and how it is used, including telemetry and reporting.
+- Where and what user data is exposed (e.g. via REST API).
+- Compliance requirements for specific user groups.
+- The content and display of a privacy policy.
+- Third-party services and advertisements.
+
+### WordPress features
+
+Since 4.9.6, WordPress provides direct support for exporting user data, removing user data, and adding a privacy policy. These do not automatically make a site compliant with a set of regulations; procedures still need to be established for response and removal timelines as well as general consideration of how data is collected and used in the first place.
+
+Another feature to keep an eye on is the REST API, in particular any meta added to the user endpoints. Be careful that exposing meta via an endpoint for the purposes of an admin feature does not inadvertently expose it publicly or to other non-admin users.
+
+### Privacy Development Best Practices
```

Comment:
This section feels informal as just one giant list.

Reply:
We have lists in some other places but they're definitely of the longer variety, i.e. with a bold one-liner and then more explanatory text. I can rework to be more like that.